lib/livebook/file_system/file.ex

```diff
@@ -31,7 +31,9 @@ defmodule Livebook.FileSystem.File do
       if path do
         resolved_path = FileSystem.resolve_path(file_system, default_path, path)
 
-        unless path == resolved_path do
+        # TODO: For `C:/Foo.txt`, the resolved path is `c:/Foo.txt`.
+        #       Is case-insensitive comparison ok? Will it be bad for case-sensitive file systems?
+        unless String.downcase(path) == String.downcase(resolved_path) do
```

Comment:
👀

Reply:
I see

https://github.com/livebook-dev/livebook/blob/0258df4a665162431f72b541333400f211d57823/lib/livebook_web/live/home_live.ex#L196-L197

This looks like a good place to normalize the path with `Path.expand`, since it comes from external location, wdyt?

Reply:
Fixed in https://github.com/livebook-dev/livebook/commit/864624ad2c25268e600e77819c63509e108de11e.